### ISSUE TITLE
[Issue] [Request] ACT Overlay lag / Hotkey hide request

### ISSUE THREAD

**Issue by vrandon21** [Styr1x/Browsingway#10]
1) Hi I just found this new plugin to use overlays in full screen, and it seems like a great idea and one much needed, but have trouble with some overlays, and just the ones I like, the Kageru one and Mopi Mopi one, I'm not sure if it's because they are web based (TBH I'm not quite sure of what I'm talking about) but I supose it because the URL for those Overlays specify a github adress. In general overlay is displayed but "lags behind" even for settings it takes a while to open them up, and sometimes it just work fine for 30 seconds, and then goes back to lagging, even the battle counter skipps seconds, goes from 5 to 10 to 13 and so on... I don't know if this is just me issue or is a general one but the "local" overlays runs fine. Also I noticed that when parser is running Dalamud Plugins window sometimes takes time to load, I'm not sure if ACT is interfering with DP, or again, is just me, but if you have any suggestion about ports or something, I would appreciate it,

2) A hotkey to hide te overlay would be awesome, since going into dalamund window to hide the overlay is just annoying, or if a hotkey is not possible, maybe a chat command or something.

Thank you for your time, and even if this is not possible, your plug-in is a much needed one.

**Comment by Styr1x** [comment]
Hi,
it does not matter if the URL is local or from github or anywhere else. Not sure why exactly you have that problem but I guess the renderer process is either crashing and restarting or it just takes a long time to render. I hope to get this fixed with the next major version of the plugin. I'll ping you once that is out so we can check if it really fixed it for you.

They are chat commands already, the settings window has an overview for them. In short `/bw inlay yourinlayname hidden on` should do the trick.

**Comment by Grammernatzi** [comment]
I also agree with being able to add hotkeys like OverlayPlugin does. I have macros to do it at the moment, but it'd be nice to be able to add hotkeys directly in the plugin itself, as that way I wouldn't need to assign it to a hotbar.
